server/controllers/LibraryController.js

```diff
@@ -492,7 +492,7 @@ class LibraryController {
     }
 
     // TODO: Create paginated queries
-    let collections = await Database.collectionModel.getOldCollectionsJsonExpanded(req.user, req.library.id, include)
+    let collections = await Database.collectionModel.getOldCollectionsJsonExpanded(req.user, req.library.id, include, 2)
```

Comment:
Instead of making this 2 at the API level, it would probably be better to make this an optional request parameter, with a default of "get everything" so 3rd party clients and scripts can still get all items from the single API call.

Not sure what a good name would be, maybe `itemCount`?

Reply:
For this one, I meant something like adding
`itemCount: req.query.itemCount && !isNaN(req.query.itemCount) ? Number(req.query.itemCount) : undefined,`
to the `payload` a few lines before, and then passing `payload.itemCount` to `getOldCollectionsJsonExpanded` instead of the constant value of 2.